dist-assets/linux/mullvad-gui-launcher.sh

```diff
@@ -8,5 +8,15 @@ else
     SANDBOX_FLAG=""
 fi
 
+SUPPORTED_COMPOSITORS="sway river"
+if [ "$XDG_SESSION_TYPE" = "wayland" ] && \
```

Comment:
If it does not become too cumbersome, our bash coding guidelines prefers `${VARIABLE:-""}` over removing `set -u`. Otherwise it allows unwanted behavior to silently sneak into other parts of the script. https://github.com/mullvad/coding-guidelines/blob/master/bash.md

Reply:
Ah yeah, sure.